appveyor.yml

```diff
@@ -23,6 +23,12 @@ install:
   - "python -m certifi >cacert.txt"
   - "set /p GIT_SSL_CAINFO=<cacert.txt"
   - "set GIT_SSL_CAINFO"
+  # Install perforce.
+  - "mkdir %TEMP%\\perforce\\bin"
+  - "pushd %TEMP\\perforce\\bin"
+  - "appveyor DownloadFile http://ftp.perforce.com/perforce/r17.2/bin.ntx86/p4.exe"
+  - "appveyor DownloadFile http://ftp.perforce.com/perforce/r17.2/bin.ntx86/p4d.exe"
```

Comment:
I don't know how big the downloads are, but maybe installing Perforce should be conditional on running the integration tests.